pkg/controller/migcluster/validation.go

```diff
@@ -72,6 +76,12 @@ func (r ReconcileMigCluster) validate(cluster *migapi.MigCluster) error {
 		return liberr.Wrap(err)
 	}
 
+	// Exposed registry route
+	err = r.validateRegistryRoute(cluster)
```

Comment:
question: is this enough? should we have a logic like this in place?

https://github.com/konveyor/mig-controller/blob/cf705e01f0dac48c3a557d61d98776b54b6c3420/pkg/controller/migstorage/provider.go#L40

Reply:
Can you explain further Alay?

Reply:
@dymurray, @shubham-pampattiwar, and I met to discuss this and decided that this is currently not needed.